packages/appstore/src/components/currency-switcher-card/demo/demo-account-card.tsx

```diff
@@ -3,19 +3,34 @@ import { observer } from 'mobx-react-lite';
 import { Button, Text } from '@deriv/components';
 import CurrencySwitcherContainer from 'Components/containers/currency-switcher-container';
 import BalanceText from 'Components/elements/text/balance-text';
-import { useStores } from 'Stores/index';
 import './demo-account-card.scss';
 import { localize } from '@deriv/translations';
+import {
+    useCFDDemoAccounts,
+    useCurrencyExchangeRate,
+    usePlatformDemoAccount,
+    useTotalAccountBalance,
+} from '@deriv/hooks';
+import { useStore } from '@deriv/stores';
 
 const DemoAccountCard = () => {
-    const { client, traders_hub } = useStores();
-    const { accounts, loginid, resetVirtualBalance } = client;
-    const { platform_demo_balance, selected_account_type } = traders_hub;
+    const { client, traders_hub } = useStore();
+    const { accounts, loginid, resetVirtualBalance, default_currency } = client;
+    const { selected_account_type } = traders_hub;
+    const platform_demo_account = usePlatformDemoAccount();
+    const cfd_demo_accounts = useCFDDemoAccounts();
+    const cfd_demo_balance = useTotalAccountBalance(cfd_demo_accounts);
+    const cfd_demo_rate = useCurrencyExchangeRate(platform_demo_account?.currency || default_currency);
 
     const canResetBalance = () => {
-        return accounts[loginid]?.balance !== 10000;
+        return loginid && accounts[loginid]?.balance !== 10000;
```

Comment:
@shayan-deriv Seems like `balance` doesn't exist on the type, Not sure if the type is wrong or the code 😬

Reply:
@farzin-deriv I checked the data and balance exists in all kind of demo accounts ( VRTCxxxxx , MFxxxxxx, and CRxxxxx) So I guess the type is wrong

Reply:
@farzin-deriv again this : `type TAccount = NonNullable<Authorize['account_list']>[0];` :smile: